curl-helper.c

```diff
@@ -915,6 +916,36 @@ static int cb_PROGRESSFUNCTION(void *data,
     return r;
 }
 
+static int cb_XFERINFOFUNCTION(void *data,
```

Comment:
this is the exact copy of PROGRESSFUNCTION callback. There must be some difference otherwise why PROGRESSFUNCTION was obsoleted

Reply:
ok, afaics it is types of values that changed, from double to curl_off_t which is integer